flocker/control/_protocol.py

```diff
@@ -90,10 +98,30 @@ class NodeStateCommand(Command):
     Used by a convergence agent to update the control service about the
     status of a particular node.
     """
-    arguments = [('node_state', NodeStateArgument())]
+    arguments = [('node_state', NodeStateArgument()),
+                 ('eliot_context', Unicode())]
     response = []
 
 
+def with_eliot_context(function):
+    """
+    Decorator for responders that accept an ``eliot_context`` argument
+    that deserializes the given Eliot context and runs the wrapped
```

Comment:
first `that` in this line is ambiguous - does it refer to the decorator or the argument?